stripe/src/main/java/com/stripe/android/StripeApiRepository.kt

```diff
@@ -889,6 +890,14 @@ internal class StripeApiRepository @JvmOverloads internal constructor(
         return StripeFileJsonParser().parse(response.responseJson)
     }
 
+    override fun retrieveObject(url: String, requestOptions: ApiRequest.Options): JSONObject {
```

Comment:
will this always be a Stripe URL?

Reply:
Yeah, we throw out any url that isn't `hooks.stripe.com` when creating the object. Possibly worth adding an additional check here?

Reply:
Yeah I'd be in favor of requiring that in the method implementation